consumer/consumererror/combine.go

```diff
@@ -15,39 +15,76 @@
 package consumererror
 
 import (
-	"fmt"
+	"errors"
 	"strings"
 )
 
+type combined []error
+
+var (
+	_ error                                    = (*combined)(nil)
+	_ interface{ Is(target error) bool }       = (*combined)(nil)
```

Comment:
nit: I don't think the `Is` and `As` interface assertions are necessary, as they just reference function declarations below.